### PR TITLE
Remove the menu from the Paywall landing page

### LINKED ISSUE #2248: Nav icons on the paywall app should be removed
The icons on the paywall app are not working (of course because the paywall app does not have an about, blog...  page)

Let's remove them.

### PR DESCRIPTION
**Pull request by cellog**
# Description

This PR removes the irrelevant menu icons from the paywall landing page.

In order to preserve the possibility of using the menu in the future, the code extracts the left-hand side of the header into a new component `HeaderTitle` and uses it in a `LeftHeader` (the header title without menu) and `Header` (the header title with menu)

Then, the `Layout` component accepts a new `showIcons` toggle that is used to determine which one to display.

Screen Shot 2019-03-21 at 10 10 44 AM
Screen Shot 2019-03-21 at 10 10 39 AM
Screen Shot 2019-03-21 at 10 10 31 AM
Screen Shot 2019-03-21 at 10 09 56 AM

# Issues

Refs #2248 

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
I think you need to make sure the same icons at the bottom do not show either...

**Comment by cellog** [comment]
yes. I'll make a 2nd PR for that.

**Comment by cellog** [comment]
new stories uploaded, PTAL @julien51 

Screen Shot 2019-03-21 at 11 19 03 AM
Screen Shot 2019-03-21 at 11 18 12 AM

**Comment by cellog** [comment]
updated the Layout stories as well:

Screen Shot 2019-03-21 at 11 34 20 AM
Screen Shot 2019-03-21 at 11 34 16 AM

File: paywall/src/components/interface/Layout.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import styled from 'styled-components'
-import Header from './Header'
+import LeftHeader from './LeftHeader'
+import FullHeader from './Header'
```
Comment (julien51) [inline paywall/src/components/interface/Layout.js]: hum. Please change the name of the import.

File: paywall/src/components/interface/HeaderTitle.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import Link from 'next/link'
+import { WordMarkLogo } from './Logo'
+import Media from '../../theme/media'
+
+export function Header({ title }) {
```
Comment (julien51) [inline paywall/src/components/interface/HeaderTitle.js]: The name of the file needs to match the name of the component.

File: paywall/src/components/interface/LeftHeader.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import HeaderTitle from './HeaderTitle'
+import Media from '../../theme/media'
+
+export default function Header({ title }) {
```
Comment (julien51) [inline paywall/src/components/interface/LeftHeader.js]: Please change the name of the component to match the name of the file

**Comment by cellog** [comment]
done. PTAL

**Pull request merged**